VireoSDK.nuspec

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package>
+  <metadata minClientVersion="2.5">
+    <id>vireo</id>
+    <version>$version$</version>
+    <title>VireoSDK</title>
+    <authors>https://github.com/ni/VireoSDK/blob/incoming/AUTHORS</authors>
+    <owners>https://github.com/ni/VireoSDK/blob/incoming/AUTHORS</owners>
+    <requireLicenseAcceptance>false</requireLicenseAcceptance>
+    <licenseUrl>https://github.com/ni/VireoSDK/blob/incoming/LICENSE.txt</licenseUrl>
+    <projectUrl>https://github.com/ni/VireoSDK/</projectUrl>
+    <description>VireoSDK provides a subset of LabVIEW runtime functionality for smaller targets.</description>
+  </metadata>
+  <files>
+    <file src="dist/Debug/esh.exe" target="content/TestFiles" />
+    <file src="vireoSDK.props" target="build" />
+  </files>
+</package>
```

Comment:
Add a newline to the end of the file to remove the red artifact that appears in online version of the diff at the end of the file.

Reply:
Done.